.github/workflows/fetch_filter_resources.yaml

```diff
@@ -159,9 +159,12 @@ jobs:
           jq -s 'map(.[])' communities/all/resources/repositories*.list_tools.json > communities/all/resources/tools.json 
           rm communities/all/resources/repositories*.list_tools.json
           rm communities/all/resources/repositories*.list_tools.tsv
-      - name: Generate wordcloud and interactive table
+      - name: Generate tools wordcloud and interactive table
         run: |
           bash sources/bin/format_tools.sh
+      - name: Generate tutorials interactive table
```

Comment:
This is supposed to be done in `sources/bin/get_community_tutorials.sh` that is run in `Filter tutorials for communities` (l 171)

Reply:
Yes, for communities, the file "tutorials.html" is generated.
The issue here is that there is no code to generate `communities/all/resources/tutorials.html`
(not community specific)

Reply:
Ah ok. The commands should probably go to `sources/bin/extract_all_tutorials.sh` instead of having an extract script

Reply:
Thanks a lot!